tcms/static/style/base.css

```diff
@@ -128,6 +128,11 @@ hr.dotted{ border:dotted #ccc 0px;border-top-width:1px;}
 .commentdelete{float:right; display:block; background-position:0 -15px;padding-left:13px; margin:-20px 5px 0 0;cursor:pointer;}
 table.noborder,table.noborder td{border:none;background:transparent;}
 
+.link {
+    color: #004481 !important;
+    text-decoration: none;
+    cursor: pointer
+}
```

Comment:
The existing CSS classes seem to be doing the job of showing the pointer just right. Why do we need another CSS style  here ?

Reply:
which classes do you mean exactly?

the existing version of this does not uses css classes, but `<a href="javascript:void(0)">` to do the job of styling

Reply:
The can't we use an `<a href="#">` tag instead of `<span>` when rendering this widget ? And no need of styling.